### DISCUSSION TITLE
M3-H5-SDK for Angular 16 available?

### DISCUSSION THREAD

**Discussion by WP-Ben** [infor-cloud/m3-h5-sdk#198] [General]
Hello,
we need an integration of this Odin framework in an Angular 16 project. Are there any plans from Infor to release an up to date version of this framework for the newer Angular versions? Or has the development of this framework been stopped?
Kind Regards,
Ben

**Comment by whernebrink** [comment]
We're currently working on a Angular 18 with ids-enterprise-ng 18 as well.
